flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/catalog/CatalogITCase.java

```diff
@@ -169,6 +186,51 @@ public void testCreateCatalogFromUserClassLoader() throws Exception {
         }
     }
 
+    @Test
+    public void testGetTablesFromGivenCatalogDatabase()
+            throws TableAlreadyExistException, DatabaseNotExistException,
+                    DatabaseAlreadyExistException {
+        final Catalog c1 = new GenericInMemoryCatalog("c1", "default");
+        final Catalog c2 = new GenericInMemoryCatalog("c2", "d2");
+
+        final CatalogManager catalogManager =
+                CatalogManagerMocks.preparedCatalogManager().defaultCatalog("c2", c2).build();
+        catalogManager.registerCatalog("c1", c1);
+        c1.createDatabase(
+                "d1",
+                new CatalogDatabaseImpl(
+                        new HashMap<String, String>() {
+                            {
+                                put("k1", "v1");
+                            }
+                        },
+                        ""),
+                true);
+        final ObjectPath path1 = new ObjectPath("d1", "t1");
+        final ObjectPath path2 = new ObjectPath(catalogManager.getCurrentDatabase(), "t2");
+
+        final TableSchema tableSchema =
```

Comment:
It would be better if we use `final Schema tableSchema =
                Schema.newBuilder().....` here.

Reply:
done